import_export/resources.py

```diff
@@ -1162,6 +1162,12 @@ def after_import(self, dataset, result, using_transactions, dry_run, **kwargs):
                 finally:
                     cursor.close()
 
+    @classmethod
+    def get_display_name(cls):
```

Comment:
Please could you add a unit test for this method because there is a line not covered.

Reply:
OK, covered that. I moved the `name` parameter to `Meta` class, as it could have collided with `name` field.